config/internal/apiserver/route.yaml.tmpl

```diff
@@ -0,0 +1,19 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: ds-pipeline-{{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    app: ds-pipeline-{{.Name}}
+    component: data-science-pipelines
+  annotations:
+    kubernetes.io/tls-acme: "true"
+spec:
+  to:
+    kind: Service
+    name: ds-pipeline-{{.Name}}
+    weight: 100
+  port:
+    targetPort: oauth
```

Comment:
[references the port identified by the service](https://github.com/HumairAK/data-science-pipelines-operator/blob/oauth-proxy/config/internal/apiserver/deployment.yaml.tmpl#L37)

> The target port on pods selected by the service this route points to. If this is a string, it will be looked up as a named port in the target endpoints port list. [[1]](https://docs.openshift.com/container-platform/4.12/rest_api/network_apis/route-route-openshift-io-v1.html)

this is a bit unclear but it's referring to port's named in the service spec